physics/ephemeris_body.hpp

```diff
@@ -371,14 +371,23 @@ bool Ephemeris<Frame>::FlowWithAdaptiveStep(
     not_null<DiscreteTrajectory<Frame>*> const trajectory,
     IntrinsicAcceleration intrinsic_acceleration,
     Instant const& t,
-    AdaptiveStepParameters const& parameters) {
+    AdaptiveStepParameters const& parameters,
+    std::int64_t const max_ephemeris_steps =
+        std::numeric_limits<std::int64_t>::max()) {
```

Comment:
Done, somehow the defaulting in the definition (which has no effect) is allowed by MSVC without warnings.